### ISSUE TITLE
Tries to download disabled / unavailable songs

### ISSUE THREAD

**Issue by GrayGalaxy** [GrayGalaxy/jiosaavn-downloader#6]
Some songs are disabled for some reason by the admin. The extension tries to download that also and throws an error... 
Thanks to @jineshpatel30 to point that out...
should be easily fixed with following response from the server

``` 
{
  ...
  "rights": {
    "code": 1,
    "reason": "Unavailable",
    "cacheable": false,
    "delete_cached_object": true
  },
  "disabled": "true",
  "disabled_text": "Unavailable",
  "cache_state": "false",
  ...
}
```

**Comment by jineshpatel30** [comment]
Errors spotted regarding this. These errors doesn't affect running download, i.e. it just skips these files in current download but due to errors, Download button doesn't show up in other playlist till we clear reported errors in the extension.

![image](https://user-images.githubusercontent.com/35566052/118164878-3af5e700-b441-11eb-8706-3c23a1732672.png)

**Comment by GrayGalaxy** [comment]
> Errors spotted regarding this. These errors doesn't affect running download, i.e. it just skips these files in current download but due to errors, Download button doesn't show up in other playlist till we clear reported errors in the extension.

It is fixed in 572fa4db10bb4d0fc1872a35e180efe9517dce03
